lib/containers/utils.pm

```diff
@@ -0,0 +1,163 @@
+# SUSE's openQA tests
+#
+# Copyright © 2020 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Basic functions for testing docker
+# Maintainer: Anna Minou <anna.minou@suse.de>, qa-c@suse.de
+
+package containers::utils;
+
+use base Exporter;
+use Exporter;
+
+use base "consoletest";
+use testapi;
+use utils;
+use strict;
+use warnings;
+use version_utils;
+
+our @EXPORT = qw(test_seccomp basic_container_tests set_up get_vars build_img test_built_img);
+
+sub test_seccomp {
+    my $no_seccomp = script_run('docker info | tee /dev/tty | grep seccomp');
+    if ($no_seccomp) {
+        my $err_seccomp_support = 'boo#1072367 - Docker Engine does NOT have seccomp support';
+        if (is_sle('<15') || is_leap('<15.0')) {
+            record_info('WONTFIX', $err_seccomp_support);
+        }
+        else {
+            die($err_seccomp_support);
+        }
+    }
+    else {
+        record_info('seccomp', 'Docker Engine supports seccomp');
+    }
+}
+
+sub basic_container_tests {
+    my $runtime = shift;
```

Comment:
```suggestion
    my $runtime = shift;
    die "You must define the runtime!" unless $runtime;
```

Reply:
Thanks Pavel, yes it's better to add this line